reference/orders.v2.oas2.yml

```diff
@@ -2635,6 +2724,39 @@ components:
               value:
                 - status: 404
                   message: The requested resource was not found.
+    orderConsignmentlineitemsResp:
```

Comment:
This schema isn't being referenced anywhere?

Reply:
You're right! I removed it.